features/step_definitions/database.rb

```diff
@@ -1,6 +1,30 @@
+require 'fileutils'
 require 'aruba/cucumber'
 require 'sqlite3'
 
+Given('that I ran {string}') do |command|
+    system command
+end
```

Comment:
I missed this before. But the convention in ruby is to use 2 spaces for indentation.

```suggestion
Given('that I ran {string}') do |command|
  system command
end
```